training/controllers/auth.py

```diff
@@ -8,14 +10,23 @@
 
 bp = Blueprint('auth', __name__)
 
+serializer = URLSafeTimedSerializer(os.environ.get('SECRET_KEY'))
```

Comment:
I've seen this in several files, can't we put it in a single place and the import it?

Reply:
Perfect!